tests/typings/webdriverio/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "outDir": "dist",
     "noImplicitAny": true,
-    "lib": ["es2018"],
+    "lib": ["es2018", "dom"],
```

Comment:
Puppeteer typings now will need dom lib to use `Document` type, possibly related to upgrading NodeJS 12 typings